index.html

```diff
@@ -1223,70 +1219,71 @@ <h2>
           human recipient is less able to process content that appears only briefly.
         </p>
         <p>
-          Information that is not currently rendered to the screen SHOULD be obscured in captures
-          unless the application has been specifically authorized to access that content (this
-          might require <a>elevated permissions</a>).
+          It is recommended that <a>user agents</a> obscure in captures such information which
```

Comment:
recommended is also RFC 2119 territory. https://infra.spec.whatwg.org/#conformance has some suggestions.

Reply:
Thanks! I've `s/recommended/encouraged/` in https://github.com/w3c/mediacapture-screen-share/pull/207